app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -541,7 +540,7 @@
     {{/ui-dropdown}}
   </div>
   <br>
-  <div class="{{unless device.isMobile 'right floated large' 'mini three'}} ui fields buttons">
+  <div class="{{unless device.isMobile 'right floated large' 'mini three'}} ui fields {{if (eq isUserVerified false) 'disabled' buttons}}">
```

Comment:
What’s ‘buttons’ doing inside the braces.

Reply:
My bad.